packages/ledger/src/lib/ledger.ts

```diff
@@ -141,6 +139,41 @@ export function setupLedger({
       }
     };
 
+    const getAccountIdFromPublicKey = async ({
+      publicKey,
+    }: GetAccountIdFromPublicKeyParams): Promise<string> => {
+      const response = await fetch("https://rest.nearapi.org/explorer", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          user: "public_readonly",
+          host: "testnet.db.explorer.indexer.near.dev",
```

Comment:
I am actually not familiar with this but will this work on a **`.near`** (mainnet) account?

Reply:
I think this documentation might help: https://github.com/near/near-indexer-for-explorer/#shared-public-access

Reply:
Now I am doing it like this so it works for testnet and mainnet.
```js
const response = await fetch(
  `${network.helperUrl}/publicKey/ed25519:${publicKey}/accounts`
);
```